webapi/src/main/twirl/queries/sparql/v1/findValueInVersions.scala.txt

```diff
@@ -58,7 +58,7 @@ WHERE {
         FILTER(?objPred != knora-base:attachedToProject)
 
         OPTIONAL {
-            ?objPred rdfs:subPropertyOf* knora-base:valueHasStandoff .
+            FILTER(?objPred = knora-base:valueHasStandoff) .
```

Comment:
Actually, the SPARQL template you're commenting on doesn't have anything to do with `ValueVersionHistoryGetRequestV1`. It's used only to check whether an update succeeded. So the existing tests that create and change values with standoff already cover it.